src/components/SkillsSection/SkillsSection.store.js

```diff
@@ -115,7 +115,7 @@ export function fetchMemberSkills (id, limit = 20) {
   return {
     type: FETCH_MEMBER_SKILLS,
     graphql: {
-      query: `query ($id: ID, $limit: Int) {
```

Comment:
Curious: does this impact code execution in some way or does it just make this a named query instead of an anonymous one?

Reply:
NM, working now...

Reply:
it just makes it named. i needed this to reference the query in the test